main.go

```diff
@@ -178,6 +189,28 @@ func main() {
 		cancel()
 	})
 
+	{
+		h := internalserver.NewHandler(
+			internalserver.WithName("Internal - thanos-rule-syncer"),
+			internalserver.WithPrometheusRegistry(registry),
+			internalserver.WithPProf(),
+		)
+
+		//nolint:exhaustivestruct
```

Comment:
We could tweak the linter config, this one is not always needed (or _almost always_ annoying and unnecessary :smile:).

Reply:
Yup, let me iterate in next PR! 🙂